src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const sum = (digit) => {
+    counter++;
+
+    const insideSum = (digit2) => {
+      if (counter < 3) {
+        return digit + digit2;
+      } else if (counter % 2 === 1) {
+        return digit + digit2;
+      } else if (counter % 2 === 0) {
+        return 'Bzzz... Error!';
```

Comment:
You can make in simpler

```suggestion
      if (counter > 3 && (counter % 2 === 0)) {
        return 'Bzzz... Error!';
      }
      
      return return digit + digit2;
```